installing/installing_ibm_z/upi/installing-ibm-z-kvm.adoc

```diff
@@ -15,15 +15,8 @@ In {product-title} version {product-version}, you can install a cluster on
 While this document refers only to {ibm-z-name}, all information in it also applies to {ibm-linuxone-name}.
 ====
 
-[IMPORTANT]
-====
-Additional considerations exist for non-bare metal platforms. Review the information in the
-link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms] before you install an {product-title} cluster.
-====
-
 == Prerequisites
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies
```suggestion
[id="prerequisites_{context}"]
== Prerequisites
```

Reply:
Was that changed I was asked to remove some releases back ...

Reply:
I am not sure about the history of the rule, sorry that inconsistent review has lead you to back-and-forth though :(